.github/workflows/test.yml

```diff
@@ -28,8 +28,8 @@ jobs:
       - name: Run Tests with Coverage
         run: |
           poetry install
-          protoc -I=$SRC_DIR --python_out=$SRC_DIR $SRC_DIR/*.proto
-          2to3 --no-diff -n -w $SRC_DIR
+          protoc -I=$PROTOBUF_SRC_DIR --python_out=$PROTOBUF_SRC_DIR $PROTOBUF_SRC_DIR/*.proto
```

Comment:
So I realized that by re-generating the proto files in CI, I'm undoing my "don't test coverage on this" edits to the generated files. 😀